backend/app/views/spree/admin/taxons/_form.html.erb

```diff
@@ -20,10 +20,12 @@
       </div>
     <% end %>
 
-    <%= f.field_container :icon do %>
-      <%= f.label :icon %><br />
-      <%= f.file_field :icon %>
-      <%= image_tag f.object.icon(:mini) if f.object.icon_present? %>
+    <% f.object.class.attachment_definitions.each do |attachment, definition| %>
```

Comment:
Doesn't this work for Paperclip only? I don't think ActiveStorage has a `attachment_definitions` class method. Maybe we should also make this partial loaded conditionally if we are using the paperclip adapter, similar to what we do for payment method partials. What do you think?